check_http_json.rb

```diff
@@ -224,8 +224,10 @@ def uri_target(options)
                 end
             end
         end
-        msg = 'Received HTTP code %s instead of 200.' % [response.code]
-        Nagios.do_exit(level.to_i, msg)
+        if level != 0
+          msg = 'Received HTTP code %s instead of 200.' % [response.code]
+          Nagios.do_exit(level.to_i, msg)
```

Comment:
```suggestion
        if not level == 0 then
            msg = 'Received HTTP code %s instead of 200.' % [response.code]
            Nagios.do_exit(level.to_i, msg)
```

Reply:
applied suggested syntax to merge request and fixed indentation